frappe/desk/query_report.py

```diff
@@ -149,33 +149,9 @@ def normalize_result(result, columns):
 
 @frappe.whitelist()
 def background_enqueue_run(report_name, filters=None, user=None):
```

Comment:
haven't broken this as some people are using this

https://sourcegraph.com/search?q=context%3Aglobal+background_enqueue_run%28&patternType=standard&sm=1&groupBy=repo